### PR TITLE
Add a script to remove users with duplicated Drupal IDs.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
This PR adds an Artisan `northstar:clean_drupal_ids` script which removes records with duplicated Drupal IDs (keeping the oldest, since that seems like a smart heuristic from spot-checking some production data).

References #334.

#### How should this be reviewed?
There's [a test](https://github.com/DFurnes/northstar/blob/0831c8380ca61960406780a15d844cf215b0d3c7/tests/Artisan/CleanDrupalIdsCommandTest.php)! Did I check all the right edge cases?

#### Checklist
- [ ] Documentation added for changed endpoints.
- [x] Tests added for new features/bug fixes.

---
For review: @angaither @weerd

### CODE REVIEW

**Comment by DFurnes** [comment]
I'm sad because this test suite passes on my local. Will investigate some more in the morn' 😢

**Comment by DFurnes** [comment]
Well, not 100% sure what's up with that, but re-installing my Composer dependencies got everything synced up with what Wercker was seeing. Still seeing some intermittent(?!?) assertion failures so thinking there might be an ordering issue in my query.

File: tests/Artisan/CleanDrupalIdsCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        $tony->setCreatedAt(new Carbon('March 10 1963'))->save();
+
+        $steve = User::create(['first_name' => 'Steve', 'last_name' => 'Rogers', 'drupal_id' => '12346']);
+        $steve->setCreatedAt(new Carbon('July 4 1920'))->save();
+
+        // Make a user with a Drupal ID, but no duplicates.
+        $kamala = User::create(['first_name' => 'Kamala', 'last_name' => 'Khan', 'drupal_id' => '55555']);
+
+        // Make some users with no Drupal ID.
+        factory(User::class, 7)->create();
```
Comment (angaither) [inline tests/Artisan/CleanDrupalIdsCommandTest.php]: The only case I think is left out of here is if `drupal_id` is set to `null` the string. I'm not sure if that matters or is actually different.

Comment (DFurnes) [inline tests/Artisan/CleanDrupalIdsCommandTest.php]: Oooh, good point. Lemme look into it.

Comment (DFurnes) [inline tests/Artisan/CleanDrupalIdsCommandTest.php]: Yep, that would've been a kablooie. 💥 Fixed in 3d9c333.

**Comment by angaither** [comment]
🚢
